src/js/utils/select.js

```diff
@@ -8,16 +14,17 @@ const isElement = value => {
  * @name select
  * @desc selects elements from the DOM by class selector or ID selector.
  * @param {string} selector - The selector to traverse the DOM with.
- * @param {HTMLElement} context - The context to traverse the DOM in.
- * @return {Array.HTMLElement} - An array of DOM nodes or an empty array.
+ * @param {Document|HTMLElement?} context - The context to traverse the DOM
+ *   in. If not provided, it defaults to the document.
+ * @return {HTMLElement[]} - An array of DOM nodes or an empty array.
  */
 module.exports = function select (selector, context) {
 
   if (typeof selector !== 'string') {
     return [];
   }
 
-  if (context === undefined || !isElement(context)) {
+  if (!context || !isElement(context)) {
```

Comment:
The check for `context === undefined` is actually unneeded here because `isElement` takes care of that case.

However, TypeScript's [strict null checking](https://ariya.io/2016/10/typescript-2-and-strict-null-checking) is not smart enough to dig into our `isElement()` function and infer its return value if `context` happens to be null or undefined, so we'll explicitly check for null and other falsy values here, ensuring that TypeScript knows exactly what's going to happen if `context` has one of those values.